kover-gradle-plugin/src/functionalTest/kotlin/kotlinx/kover/gradle/plugin/test/functional/cases/LoggingTaskTests.kt

```diff
@@ -45,7 +45,7 @@ internal class LoggingTaskTests {
         run(":koverLog") {
             checkOutcome("koverPrintCoverage", "SUCCESS")
             taskOutput("koverPrintCoverage") {
-                assertEquals("application line coverage: 57.1429%\n\n", this)
+                assertContains(this, "application line coverage: 57.1429%\n\n")
```

Comment:
Is this change related? Why changing artifact attribute adds something to `koverPrintCoverage` output?

Reply:
On some Gradle versions function `taskOutput` works incorrectly (has extra output). 
Will change it separately.